modules/CurveAnalysis/src/server/renderCurve.R

```diff
@@ -28,18 +28,17 @@ renderCurve <- function(getParams, postData) {
   # Get protocol curve min and max
   protocol_display_values <- racas::get_protocol_curve_display_min_and_max_by_curve_id(parsedParams$curveIds[[1]])
 
-  output <- applyParsedParametersToFitData(fitData, parsedParams, protocol_display_values)
-  data <- output$data
-  parsedParams <- output$parsedParams
-
-  fitData[ , renderingOptions := list(list(get_rendering_hint_options(renderingHint))), by = renderingHint]
-
-  data <- list(parameters = as.data.frame(fitData), points = as.data.frame(rbindlist(fitData$points)))
```

Comment:
The main issue was here.  Basically wiping out the data variable which eliminated any of the work applied in `applyParsedParmatersToFitData`